modules/apps/headless/content-space/content-space-headless-apio-impl/bnd.bnd

```diff
@@ -0,0 +1,4 @@
+Bundle-Name: Liferay Content Space Headless Apio Implementation
+Bundle-SymbolicName: com.liferay.content.space.headless.apio.impl
+Bundle-Version: 2.0.0
```

Comment:
Does it makes sense to have both packages now? We have to discuss it.

Reply:
We'll go with two modules as discussed